src/main/java/io/vlingo/schemata/infra/persistence/SchemataObjectStore.java

```diff
@@ -0,0 +1,210 @@
+                                SqlStatement::bindFields),
+                        new UnitStateMapper());
+
+        mappersLookup.put(UnitState.class, unitStateMapper);
+
+        StateObjectMapper contextStateMapper =
+                StateObjectMapper.with(
+                        ContextState.class,
+                        JdbiPersistMapper.with(
+                                "INSERT INTO CONTEXT(contextId, unitId, organizationId, namespace, description) " +
+                                        "VALUES (:contextId.value, :contextId.unitId.value, " +
+                                        ":contextId.unitId.organizationId.value, :namespace, :description)",
+                                "UPDATE CONTEXT SET namespace = :namespace, description = :description WHERE id = :id",
+                                SqlStatement::bindFields),
+                        new ContextStateMapper());
+
+        mappersLookup.put(ContextState.class, contextStateMapper);
+
+        StateObjectMapper schemaStateMapper =
+                StateObjectMapper.with(
+                        SchemaState.class,
+                        JdbiPersistMapper.with(
+                                "INSERT INTO SCHEMA(schemaId, contextId, unitId, organizationId, category, name, description) " +
+                                        "VALUES (:schemaId.value, :schemaId.contextId.value, :schemaId.contextId.unitId.value, " +
+                                        ":schemaId.contextId.unitId.organizationId.value, :category, :name, :description)",
+                                "UPDATE SCHEMA SET category = :category, name = :namespace, description = :description WHERE id = :id",
+                                SqlStatement::bindFields),
+                        new SchemaStateMapper());
+
+        mappersLookup.put(SchemaState.class, schemaStateMapper);
+
+        StateObjectMapper schemaVersionStateMapper =
+                StateObjectMapper.with(
+                        SchemaVersionState.class,
+                        JdbiPersistMapper.with(
+                                "INSERT INTO SCHEMAVERSION(schemaVersionId, schemaId, contextId, unitId, " +
+                                        "organizationId, description, specification, status, versionState) " +
+                                        "VALUES (:schemaVersionId.value, :schemaVersionId.schemaId.value, " +
+                                        ":schemaVersionId.schemaId.contextId.value, :schemaVersionId.schemaId.contextId.unitId.value, " +
+                                        ":schemaVersionId.schemaId.contextId.unitId.organizationId.value, :description, " +
+                                        ":specification.value, :status.value, :versionState.value)",
+                                "UPDATE SCHEMAVERSION SET description = :description, specification = :specification.value, " +
```

Comment:
@VaughnVernon For all the nested objects like here (SchemaVersion, Schema, Context, Unit) it doesn't take into account that one can update a nested id and change the hierarchy. I don't know if this is necessary but I just wanted to highlight it.